tests/functional/cli/test_factory.py

```diff
@@ -97,3 +97,21 @@ def test_filename_and_lineno_included_in_syntax_error(clifactory):
     message = str(excinfo.value)
     assert 'app.py' in message
     assert 'line 1' in message
+
+
+def test_can_import_vendor_package(clifactory):
+    # Tests that vendor packages can be imported during config loading.
+    vendedlib_dir = os.path.join(clifactory.project_dir, 'vendor', 'vendedlib')
+    os.makedirs(vendedlib_dir)
+    open(os.path.join(vendedlib_dir, '__init__.py'), 'a').close()
+    with open(os.path.join(vendedlib_dir, 'submodule.py'), 'a') as f:
+        f.write('CONST = "foo bar"\n')
+    app_py = os.path.join(clifactory.project_dir, 'app.py')
+    with open(app_py, 'r+') as f:
```

Comment:
This doesn't need to block merging, but wouldn't just be simpler to open the file at the end and append `from vendedlib import submodule\napp.imported_value = submodule.CONST`?  Not really sure if the `.seek(0, 0)` and `.seek(0, 1)` make the test any clearer.